core/src/main/scala/filodb.core/reprojector/MapDBMemTable.scala

```diff
@@ -22,32 +22,17 @@ import filodb.core.metadata.{Column, Dataset}
  * NOTE: MapDB apparently has an issue with memory leaks, deleteTable / deleteRows does not appear to
  * compact the table properly.
  *
- * MapDB table namespacing scheme:
- *   datasetName/version/active
- *   datasetName/version/locked
- *
  * ==Configuration==
  * {{{
  *   memtable {
- *     backup-dir = "/tmp/filodb.memtable/"
- *     max-rows-per-table = 1000000
- *     min-free-mb = 512
  *   }
  * }}}
  */
-class MapDBMemTable(config: Config) extends MemTable {
-  import MemTable._
-  import RowReader._
+class MapDBMemTable[K](val projection: RichProjection[K], config: Config) extends MemTable[K] {
   import collection.JavaConversions._
```

Comment:
Sorry, not sure what you mean exactly?  I prefer to place wildcard imports in a smaller scope, though I don't follow that rule universally myself.